### PR TITLE
feat(focus): init

### PR DESCRIPTION
**Pull request by IncognitoTGT**
## 🎉 Theme for Focus 🎉

Focus is a Student information System used by schools to manage student information, such as grades, classes, and enrollment.

## 💬 Additional Comments 💬

CSS classes are really weird and I ended up using a lot of !important, and I had to make an outline for the background to extend on the homepage.

Also, the preview image has things blurred as using the font did not apply everywhere, as the site's css uses !important way more than it should.

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - [x] `catppuccin.user.css` - all the CSS for the userstyle, based on the
        template.
  - [x] `preview.webp` - composite image of all four individual flavor screenshots (taken with the default accent color of mauve) stitched together, generated via [Catwalk](https://github.com/catppuccin/catwalk).

### CODE REVIEW

File: scripts/userstyles.yml [hunk 1]
```diff
@@ -325,6 +327,13 @@ userstyles:
     readme:
       app-link: "https://www.freedesktop.org"
     current-maintainers: [*NK308]
+  focus:
+    name: Focus SIS
+    categories: [userstyle, education, productivity]
```
Comment (uncenter) [inline scripts/userstyles.yml:332]:
```suggestion
    categories: [education, productivity]
```

File: scripts/userstyles.yml [hunk 2]
```diff
@@ -325,6 +327,13 @@ userstyles:
...
+    color: peach
+    readme:
+      app-link: "focusschoolsoftware.com"
```
Comment (uncenter) [inline scripts/userstyles.yml:335]:
```suggestion
      app-link: "https://focusschoolsoftware.com"
```

File: scripts/userstyles.yml [hunk 3]
```diff
@@ -558,7 +571,7 @@ userstyles:
     icon: musicbrainz
     color: peach
     readme:
-      app-link: 'https://listenbrainz.org'
+      app-link: "https://listenbrainz.org"
```
Comment (uncenter) [inline scripts/userstyles.yml:574]: Please don't modify these entries.

File: styles/focus/catppuccin.user.css [hunk 4]
```diff
@@ -0,0 +1,538 @@
...
+		@overlay0: #6c7086;
+		@surface2: #585b70;
+		@surface1: #45475a;
+		@surface0: #313244;
+		@base: #1e1e2e;
+		@mantle: #181825;
+		@crust: #11111b;
+	}
+	;
+}
```
Comment (uncenter) [inline styles/focus/catppuccin.user.css:536]: Please don't format this (hence the prettier ignore comment above this block).

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by IncognitoTGT** [comment]
[242b5c9](https://github.com/catppuccin/userstyles/pull/1205/commits/242b5c9768e0ec3dc82b937aae01bdfda46097a9)

specifically the table:

image
corner buttons are really weird too. who thought it was a good idea to use a table element for layout 🤦

File: styles/focus/catppuccin.user.css [hunk 5]
```diff
@@ -0,0 +1,441 @@
...
+@description Soothing pastel theme for Focus SIS
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+@-moz-document regexp('^(https?:\\/\\/)?([a-zA-Z0-9-]+\\.)+focusschoolsoftware\\.com(\\/.*)?$') {
```
Comment (uncenter) [inline styles/focus/catppuccin.user.css:17]:
```suggestion
@-moz-document domain('focusschoolsoftware.com') {
```
Should have the same behavior if I'm reading this regex correctly.

Comment (IncognitoTGT) [reply styles/focus/catppuccin.user.css:17]: focusschoolsoftware.com is completely different from the SIS and it won't apply there. The regex i put only works on subdomains of it, which is how the SIS is setup by most schools.

**Comment by IncognitoTGT** [comment]
ok what did i just do

**Pull request closed without merging**